Sources/LeafKit/LeafLexer/LeafLexer.swift

```diff
@@ -195,9 +195,12 @@ internal struct LeafLexer {
         // Test for Operator first - this will only handle max two character operators, not ideal
         // Can't switch on this, MUST happen before trying to read tags
         if current.isValidOperator {
-            var op = Operator(rawValue: String(current) + String(src.peek()!))
-            if op == nil { op = Operator(rawValue: String(current)) } else { src.pop() }
-            if op != nil { return .parameter(.operator(op!)) }
+            // Try to get a valid 2char Op
+            var op = LeafOperator(rawValue: String(current) + String(src.peek()!))
+            if op != nil, !op!.available { throw LeafError(.unknownError("\(op!) is not yet supported as an operator")) }
+            if op == nil { op = LeafOperator(rawValue: String(current)) } else { src.pop() }
+            if op != nil, !op!.available { throw LeafError(.unknownError("\(op!) is not yet supported as an operator")) }
+            return .parameter(.operator(op!))
```

Comment:
Little ugly but new operators are in the definitions but not yet implemented, this guards with a more informative message in case someone tries to `#(10 % 3)` or `#(variable = 10)`